screens/Profile.tsx

```diff
@@ -330,6 +331,38 @@ export default function ProfileScreen({
     );
   }, [colorScheme, logout]);
 
+  const showRevokeActionSheet = useCallback(async () => {
```

Comment:
Totally doesn't have to happen in this PR but might make sense to show the number of other installations. Like a `revoke 10 other installations`.

Reply:
This can be retrieved from the new `InboxState`